components/x-gift-article/src/HighlightRecipientMessage.scss

```diff
@@ -5,56 +5,52 @@
 @include oTypography();
 
 .x-gift-article__highlight-recipient-message {
-    position: relative;;
-    background-color: oColorsByName('slate');
-    color: oColorsMix('ft-grey', 'white', 20);
-    padding: oSpacingByName('s3');
-  
-    .x-gift-article__highlight-recipient-message__title {
-      @include oTypographySans($scale: 1, $weight: 'regular');
-      margin: 0;
-      color: oColorsByName('white');
-    }
-  
-    .x-gift-article__highlight-recipient-message__description {
-      @include oTypographySans($scale: 0);
-      margin: 0;
-    }
-    
-    .x-gift-article__highlight-recipient-message__divider{
-      padding-top: oSpacingByName('s2');
-      padding-bottom: oSpacingByName('s3');
-      &::after {
-        border-top: oSpacingByName('s1') solid oColorsByName('mint');
-        content: "";
-        position: absolute;
-        width: 40px;
-        content: "";
-        display: block;
-      }
-    }
-  
-    .x-gift-article__highlight-recipient-message__close {
-      background-image: url('https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1%3Across?source=editor&tint=%23FFFFFF&bgcolor=%23FFFFFF');
-      background-repeat: no-repeat;
-      background-size: contain;
-      background-position: 50%;
-      background-color: transparent;
-      width: oSpacingByName('s6');
-      height: oSpacingByName('s6');
-      border: none;
-      padding: 0;
-      content: "";
-      position: absolute;
-      right: oSpacingByName('s2');
-      top: oSpacingByName('s2');
-    }
-  
-    .x-gift-article__highlight-recipient-message__save-highlights-button {
-      background: none;
-      border: none;
-      padding: 0;
-    }
-}
+	position: relative;
+	background-color: oColorsByName('slate');
+	color: oColorsMix('ft-grey', 'white', 20);
+	padding: oSpacingByName('s3');
+
+	.x-gift-article__highlight-recipient-message__title {
```

Comment:
**Change request:**** The classnames used through out this PR still don't conform to the BEM naming convention. This currently reads as `BLOCK__ELEMENT__ELEMENT` Maybe something like this would better match BEM `x-gift-article-highlight-recipient-message__title`

Reply:
For more context: https://getbem.com/naming/ . So yes I think we should precede each of the elements with (`__title`,` __description`,` __divider`) with `x-gift-article-highlight-recipient-message`